packages/views/screens/managers/wallet/overview.js

```diff
@@ -64,10 +65,10 @@ const Overview = ({
     state.bookmarks[activeToken].find((item) => item.address === address));
   const host = useSelector(
     (state) =>
-      (state.account
-        && state.account.info
-        && state.account.info[activeToken]
-        && state.account.info[activeToken].summary?.address)
+      (state.wallet
+        && state.wallet.info
+        && state.wallet.info[activeToken]
+        && state.wallet.info[activeToken].summary?.address)
```

Comment:
its the same but much easer to read
```suggestion
      ( state?.wallet?.info?[activeToken]?.summary?.address)
```